hca-foundation/juniorachievement
# Tb forms update

# Description
**New**
- Added a handler function for the 'personalFinanceData' property on state so that answers from AboutMe, AboutMyFuture, and AboutMyFacilitators are properly updating state.

**Fix**
- Added conditional to render 'Submit' button(instead of Next) on the AboutMyFuture component if the user is taking the preassessment.
- Updated DataManager's 'GET' to accept a 'tab' parameter so it can be used for assessment visualizations later
- Corrected minor typos
- Redirect to /completionpage stopped working, so used props.history.push() to redirect to completion page

**Chore**
- Removed unused useRefs and innerRefs in AboutMe, AboutMyFuture, and AboutMyFacilitators


# Testing Instructions
- Ensure pre-test's AboutMyFuture page renders a 'Submit' button instead of 'Next'
- Ensure AboutMyFacilitators only renders on the post-test
- Open the React inspector in your browser and ensure state is updating correctly(personalFinanceData property should be updating as radio buttons are clicked; ex: q25_answer: "2", q26_answer: "1")
- Respond to all questions on AboutMe, AboutMyFuture, and AboutMyFacilitators(if on posttest) and hit submit to ensure you're redirected to the /completionpage
